### DISCUSSION TITLE
Inhaled Compounds

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#799] [Questions & Problems] [answered]
Is it possible to introduce inhaled compounds in the alveolar space of lungs with uptake into venous plasma in PK-Sim?

**Comment by StephanSchaller** [comment, marked as answer]
Hi @pburkat, not by default.
But there is some work on Lung absorption: 

https://github.com/Open-Systems-Pharmacology/Inhalation-model
https://github.com/Open-Systems-Pharmacology/Lung-absorption-model

Best,
Stephan

**Comment by pburkat** [comment]
Thank you, much appreciated.

Best,
Paul
